### DISCUSSION TITLE
I don't get it - a journey

### DISCUSSION THREAD

**Discussion by ghost** [1000001101000/Debian_on_Buffalo#89] [Q&A]
**Got a LS-XL. Ver. 1.75 installed. I follow 'Example: Loading installer files using stock firmware'**
$ java -jar acp_commander_cli.jar -t 192.168.1.131 -pw mypassword -o  **< note I used acp_commander.jar since there is no _cli**
                                                                                         **< it responds admin passord? so I give it again**
<$ cd ..
<$ cd Buster/installer_images/
<$ python -m SimpleHTTPServer 8000 >/dev/null 2>&1 &     **< don't do any of this yet just want to see whats going on first**
<[1] 31817 
$ telnet 192.168.1.131
Trying 192.168.1.131...

**No telnet installed**
**OK so I turn on SSH via  acp_commander_gui.jar**

<[admin@LS220DED26 ~]$ cd /boot   **< so far so good (of course I use root and it returns LS-XL27E**
<[admin@LS220DED26 boot]$ mv uImage.buffalo uImage.buffalo.bak 
<[admin@LS220DED26 boot]$ mv initrd.buffalo initrd.buffalo.bak

**I then use sftp instead of wget**
**I try using Stretch because Buster doesn't have a LS-XL uImage file**

<[admin@LS220DED26 boot]$ mv uImage.buffalo.ls220d uImage.buffalo
<[admin@LS220DED26 boot]$ sync
<[admin@LS220DED26 boot]$ exit
<logout
<Connection closed by foreign host.

< $ java -jar acp_commander_cli.jar -t 192.168.1.131 -pw mypassword -c "reboot"  **< and again it asks the password again**

**And nothing - no reboot SO I use acp_commander_gui.jar and send the reboot from that**

**And then endless reboots**
**And I would really like to have Buster on it but I must be doing something wrong.**
**I have a LS-WXL that has 1.74 on it and it basically does the same thing but right now its holding the back ups of the LS-XL**

**ANY comments will be appreciated**

**Comment by 1000001101000** [comment]
Good to hear from you!

I think I know why you're having trouble with the LS-XL, it doesn't actually work with the installer images anymore. The problem is that it only has 64MB of ram which for some reason isn't enough for the installer provided by Debian anymore. I set up an alternate process that directly generates a disk image which is currently the only way to get Debian running on this model:
https://github.com/1000001101000/Debian_on_Buffalo/wiki/Alternate-install-method-via-debootstrap-script

The LS-WXL should work just fine with the armel installer, it's one of the main devices I've used for testing that installer. I suspect the problem you're having with that device is different in some way.

**Reply by ghost** [reply]
Well that sounds bad... Guess I'll take the LS-XL back to stock 1.75. I mean it works fine there, I'm not looking to build a disk image for it. Then I'll put the stuff from the LS-WXL back on it and try again with that one. Saw a ton of duplicate files moving them so will clean those out too.
The LS-WXL has been down for quite a while. I bought two 1TB drives and had trouble putting the stock 1.74 back on it. I do have a copy of 1.40. Would that be easier to convert?
I would like to put 4TB on the LS-WXL.
Oh well Tanxs fo the help. Will come back with either more or less...
